modules/ossm-federation-config-failover.adoc

```diff
@@ -0,0 +1,180 @@
+
+You configure Federation for failover by setting the `importAsLocal` and `locality` settings in a `ServiceImportSet` resource and then configuring a `DestinationRule` that configures failover for the service to the locality specified in the `ServiceImportSet`.
+
+.Prerequisites
+
+* Two or more OpenShift Container Platform 4.6 or above clusters already networked and federated.
+* `ExportServiceSet` resources already created for each mesh peer in the federated mesh.
+* `ImportServiceSet` resources already created for each mesh peer in the federated mesh.
+* An account with the cluster-admin role.
+
+== Configuring an ImportServiceSet for failover
+
+Locality-weighted load balancing allows administrators to control the distribution of traffic to endpoints based on the localities of where the traffic originates and where it will terminate. These localities are specified using arbitrary labels that designate a hierarchy of localities in {region}/{zone}/{sub-zone} form.
+
+In the examples in this section, the `green-mesh` is located in the `us-east` region, and the `red-mesh` is located in the `us-west` region.
+
+.Example ImportServiceSet resource from red-mesh to green-mesh
+[source,yaml]
+----
+kind: ImportedServiceSet
+apiVersion: federation.maistra.io/v1
+metadata:
+  name: red-mesh #name of mesh that exported the service
+  namespace: green-mesh-system #mesh namespace that service is being imported into
+spec:
+  importRules: # first matching rule is used
+  # import ratings.bookinfo as ratings.bookinfo
+  - type: NameSelector
+    importAsLocal: true
+    nameSelector:
+      namespace: bookinfo
+      name: ratings
+      alias:
+        # service will be imported as ratings.bookinfo.svc.red-mesh-imports.local
+        namespace: bookinfo
+        name: ratings
+  #Locality within which imported services should be associated.
+  locality:
+    region: us-west
+----
+
+.Table ImportedServiceLocality Fields
+|===
+| Name | Description | Type
+
+|region:
+|Region within which imported services are located.
+|string
+
+|subzone:
+|Subzone within which imported services are located.  I Subzone is specified, Zone must also be specified.
+|string
+
+|zone:
+|Zone within which imported services are located.  If Zone is specified, Region must also be specified.
+|string
+|===
+
+
+.Procedure
+
+. Log in to the {product-title} CLI as a user with the `cluster-admin` role. Enter the following command.
++
+[source,terminal]
+----
+$ oc login --username=<NAMEOFUSER> <API token> https://<HOSTNAME>:6443
+----
++
+. Change to the project where you installed the control plane.
++
+[source,terminal]
+----
+$ oc project <smcp-system>
+----
++
+For example, `green-mesh-system`.
++
+[source,terminal]
+----
+$ oc project green-mesh-system
+----
++
+. Run the following command to edit the `ImportServiceSet` file, where `<ImportServiceSet.yaml>` includes a full path to the file you want to edit:
```

Comment:
To maintain some consistency with the above procedure maybe rework this to include ",enter the following command"? Something like this?